src/main/java/io/conduit/SpecifierService.java

```diff
@@ -41,27 +41,48 @@ public class SpecifierService extends SpecifierPluginGrpc.SpecifierPluginImplBas
     @Override
     public void specify(Request request, StreamObserver<Response> responseObserver) {
         responseObserver.onNext(
-                Response.newBuilder()
-                        .setName("s3-iceberg")
-                        .setSummary("An S3 destination plugin for Conduit, written in Java.")
-                        .setVersion("v0.1.0")
-                        .setAuthor("Meroxa, Inc.")
-                        .putAllDestinationParams(buildDestinationParams())
-                        .build()
+            Response.newBuilder()
+                .setName("s3-iceberg")
+                .setSummary("An S3 destination plugin for Conduit, written in Java.")
+                .setVersion("v0.1.0")
+                .setAuthor("Meroxa, Inc.")
+                .putAllDestinationParams(buildDestinationParams())
+                .build()
         );
         responseObserver.onCompleted();
     }
 
     private Map<String, Specifier.Parameter> buildDestinationParams() {
-        // todo add all params
         Map<String, Parameter> params = new HashMap<>();
-        params.put("aws.accessKeyId", Specifier.Parameter.newBuilder()
-                .setDescription(
-                "AWS access key id.")
+        params.put(
+            "catalog.name",
+            Specifier.Parameter.newBuilder()
+                .setDescription("Catalog name")
+                .setDefault("")
+                .setType(Specifier.Parameter.Type.TYPE_STRING)
+                .addValidations(requiredValidation)
+                .build()
+        );
+        params.put(
+            "namespace",
+            Specifier.Parameter.newBuilder()
+                .setDescription("Namespace")
                 .setDefault("")
                 .setType(Specifier.Parameter.Type.TYPE_STRING)
-//                .addValidations(requiredValidation)
-                .build());
+                .addValidations(requiredValidation)
+                .build()
+        );
+        params.put(
+            "table.name",
+            Specifier.Parameter.newBuilder()
+                .setDescription("Table name")
+                .setDefault("")
+                .setType(Specifier.Parameter.Type.TYPE_STRING)
+                .addValidations(requiredValidation)
+                .build()
+        );
+
+        // todo make Catalog URI required
```

Comment:
what about the s3 params? shouldn't we have them here too?

Reply:
You're right!